### DISCUSSION TITLE
TShock 4.5.16 for Terraria 1.4.3.6

### DISCUSSION THREAD

**Discussion by hakusaro** [Pryaxis/TShock#2591] [Show and tell]
## Notable changes
* Added preliminary support for Terraria 1.4.3.6. (@SignatureBeef, @hakusaro)

## Upgrading
To upgrade TShock, replace all of the files in the location of your server installation with all of the files from the release zip we provide. Overwrite all files. No data will be lost.

<hr /><em>This discussion was created from the release <a href='https://github.com/Pryaxis/TShock/releases/tag/v4.5.16'>TShock 4.5.16 for Terraria 1.4.3.6</a>.</em>

**Comment by wwa171** [comment]
Thank you for always

**Reply by hakusaro** [reply]
You’re welcome!!
